css/properties/position.json

```diff
@@ -44,6 +44,38 @@
             "deprecated": false
           }
         },
+        "absolute": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
```

Comment:
The parent has notes about the absolute value in Firefox that should now live with the value entry.